crates/swc_ecma_codegen/src/config.rs

```diff
@@ -11,13 +13,16 @@ pub struct Config {
     /// e.g. `const foo = 1;` with [EsVersion::Es3] will emitted as `const foo =
     /// 1` without verification.
     /// This is because it's not a concern of the code generator.
+    #[serde(default)]
```

Comment:
This should be `EsVersion::latest()`  by default

Reply:
@kdy1 Fixed :+1: